data/exploratory-data-analysis/summarize-represent-and-interpret-data-on-a-single-count-or-measurement-variable/mean-median-mode.md

```diff
@@ -0,0 +1,63 @@
+# Insight Title
+author: claremarie
+
+levels:
+
+  - basic
+
+type: normal
+
+inAlgoPool: false
+
+category: fundamental
+
+tags:
+
+  - central-tendency
+  - statistics
+
+notes: 'this is my first attempt, so bear with me'
+
+
+
+---
+## Content
+
+One of the first questions you should ask when encountering numerical data is: "What is a typical value for this dataset?"
+
+
+---
+---
+## Practice
+
+Considering the list of integers
```

Comment:
This question is great, perfect distinction between practice and revision question